script/bootstrap

```diff
@@ -103,11 +103,11 @@ install_packages_brew()
     echo 'Installing pretty tools useful for code contributions...'
 
     # add clang-format for pretty
-    CLANG_FORMAT_VERSION="clang-format version 10"
+    CLANG_FORMAT_VERSION="clang-format version 9"
     command -v clang-format-9 || (command -v clang-format && (clang-format --version | grep -q "${CLANG_FORMAT_VERSION}")) || {
         brew install llvm@9
         sudo ln -s "$(brew --prefix llvm@9)/bin/clang-format" /usr/local/bin/clang-format-9
-    }
+    } || echo 'Failed to install llvm@9. Install it manually if you need.'
```

Comment:
How about using the same "warning" message as above? "Failed" can be interpreted by a user as needing to fix it.

Reply:
fixed